curvefs/src/metaserver/trash.cpp

```diff
@@ -123,11 +123,7 @@ bool TrashImpl::NeedDelete(const TrashItem &item) {
                      << ", ret = " << MetaStatusCode_Name(ret);
         return false;
     }
-    if (inode.has_openmpcount() && inode.openmpcount() > 0) {
-            return false;
-    } else {
-        return ((now - item.dtime) >= options_.expiredAfterSec);
-    }
+    return ((now - item.dtime) >= options_.expiredAfterSec);
```

Comment:
if delete opencount, when deleting an open file, what should trash do here?

Reply:
When the file arrives in the trash, it must be deleted by the user, so it is good to delete it directly. The current curvefs does not guarantee the correctness of one writing and one deleting in the case of multiple mounts.

As for what to keep the file after the user deletes it by mistake, it depends on what the recycle bin does.

Reply:
The recycle bin only gives users a chance to retrieve files under specified conditions, and does not guarantee that they can be retrieved.